docs/create/mapping.md

```diff
@@ -112,3 +112,156 @@ Due to the limitations of the virtual machine in our sandbox, currently we only
 If a library is depends on any modules in **ESM** format, the virtual machine will **NOT** compile and return the outcome. 
 
  
+## Specific substrate chain
+
+The custom types/interfaces from a specific substrate chain (other than polkadot/kusam) are usually difficult for the user to import and implement. And standard API injected from the sandbox is not support query or RPC call on specific module/method.
```

Comment:
user can use standard api to query, they just didn't get a proper typescript definition.

Reply:
Is the new description valid?